src/js/yeldaWebchatInjector.js

```diff
@@ -277,6 +279,43 @@ class YeldaChat {
     return url
   }
 
+  /**
+   * Add 'y_active' class to the webChatContainer if required
+   *
+   * To control the opacity of the iframe container (.yelda_iframe_container) and the assistant image (.yelda_assistant_img),
+    * We add and remove the "y_active" CSS class to the webChatContainer (.yelda_container) which is always visible
+    * By default
+    * - yelda_iframe_container is hidden (opacity: 0;)
+    * - assistant image is visible (opacity: 1 for yelda_assistant_img)
+    *
+    * When we add the class "y_active" to  (webChatContainer)
+    * - yelda_iframe_container is now visible (.y_active .yelda_iframe_container => opacity: 1)
+    * - the assistant image is hidden (.yelda_container.y_active .yelda_assistant_img => opacity: 0
+    *
+    * @param {Boolean} data.shouldBeOpened
+    * @param {Boolean} data.isStartBtn
+    * @param {Boolean} data.canBeClosed
+    */
+  initWebChatContainerYActiveClass(data) {
+      // should be opened on load OR has a start button
+      const shoulBeOpened = data.shouldBeOpened || data.isStartBtn
+
+      // cannot be closed (canBeClosed explicitly set to false)
+      const cantBeClosed = data.hasOwnProperty('canBeClosed') && !data.canBeClosed
+
+      /**
+       * If we are sure that the webchat should be opened from the creation and displayed all the time
+       * we add the y_active class to webChatContainer at the creation
+       *
+       * Otherwise yelda_assistant_img click event management will take care of showing and hiding the webchat
+       * - closeChat() remove "y_active" from the yelda container
+       * - openChat() add y_active" to the yelda container
+       */
+     if (shoulBeOpened && cantBeClosed) {
+       this.webChatContainer.classList.add('y_active')
```

Comment:
class names can be moved to config

Reply:
indeed, good suggestion.
multiple classes are manipulated in the injector, so i instead open an issue on which we could work on when will we have more time.
https://github.com/Yeldaai/chat-plugin/issues/112